src/constructs/core/parameters/s3.test.ts

```diff
@@ -7,10 +7,10 @@ describe("GuDistributionBucketParameter", () => {
   it("can exist in multiple GuStacks within the same App", () => {
```

Comment:
"App" here is a CDK [`App`](https://docs.aws.amazon.com/cdk/v2/guide/apps.html#apps_construct) (aka the root of the tree), not a Guardian app (aka a string identifier).

Reply:
Thanks for disambiguating; perhaps we should update the test name to `"can exist in multiple GuStacks within the same CDK App"`?

Reply:
Yeah, I wonder if we should think about better ways to disambiguate 'app' and 'stack' in CDK somehow, as the terms are overloaded. (Not as part of this PR though.)

Reply:
Will leave this for a future PR (not least because this will change again when we upgrade to CDK v2).